README.md

```diff
@@ -15,37 +15,64 @@
 
 The rust API is also sufficiently flexible to enable you to code up a computational graph and resulting circuit from scratch. For examples on how to do so see the **library examples** section below.
 
-In the backend we use [Halo2](https://github.com/privacy-scaling-explorations/halo2) as a proof system.  
+In the backend we use [Halo2](https://github.com/privacy-scaling-explorations/halo2) as a proof system.
 
-For more details on how to use `ezkl`, see below ! 
+For more details on how to use `ezkl`, see below !
 
 ----------------------
 
 ## Contributing 🌎
 
-If you're interested in contributing and are unsure where to start, reach out to one of the maintainers:  
+If you're interested in contributing and are unsure where to start, reach out to one of the maintainers:
 
 * dante (alexander-camuto)
 * jason ( jasonmorton)
 
-More broadly: 
+More broadly:
 
-- Feel free to open up a discussion topic to ask questions. 
+- Feel free to open up a discussion topic to ask questions.
 
-- See currently open issues for ideas on how to contribute. 
+- See currently open issues for ideas on how to contribute.
 
-- For PRs we use the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) naming convention. 
+- For PRs we use the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) naming convention.
 
 ----------------------
 
 ## Getting Started ⚙️
 
+### building the project 🔨
 Note that the library requires a nightly version of the rust toolchain. You can change the default toolchain by running:
 
 ```bash
-rustup override set nightly         
+rustup override set nightly
 ```
 
+After which you may build the library
+
+```bash
+cargo build --release
+```
+
+A folder `./target/release` will be generated. Add this folder the your PATH environment variable to call `ezkl` from CLI.
```

Comment:
Small typo -> "add this folder _to_" + "_the_ CLI"

Reply:
Fixed the typo